contents/handbook/people/spending-money.md

```diff
@@ -63,7 +63,8 @@ PostHog will provide you with office equipment. Please note that it remains Post
 
 We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you. We'd prefer everyone uses Apple laptops, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier, not many.
 
-* If you are in a technical or design role, we recommend a Macbook Pro with an Apple Silicon processor and at least 16GB of RAM.
+* If you are in an engineering role, we recommend a Macbook Pro with an Intel processor with at least 32GB of RAM. The processor selection here is important as we want to ensure that you're able to run all the technologies in our stack and several of them have yet to be adapted on the new Apple architecture.
```

Comment:
@mariusandra tagging you to provide some thoughts on how much memory is actually needed here because if i remember correctly you develop by spinning up the entire ee stack locally

Reply:
Fwiw I have a 16GB and it's not enough. Can get stuck quite easily when developing.

Reply:
Makes sense. 32gb it is

Reply:
32GB should be enough, though on personal machines I always max it out. 64GB is what I have, and here's my setup at the end of the day, while running all the clickhouse stuff:

![image](https://user-images.githubusercontent.com/53387/116578852-1adb0980-a912-11eb-9c20-6943317ad4a2.png)

32GB should be enough though.

Reply:
+1 on 32gb min requirement

Reply:
🤮

Reply:
_cries in 16gb_